docs/advanced-usage/user-quotas/index.md

```diff
@@ -116,21 +116,44 @@ you will not free up any space:
 
 ### Prune the workspace
 
-Another quick way to free some storage from your workspace is using the `reana-client prune`
-command. This command deletes all the files from the workspace of the specified workflow
-that are neither input nor output files, which can free up a significant amount of space.
+Another quick way to free some storage from your workspace is using the
+`reana-client prune` command. This command deletes all the files from the
+workspace of the specified workflow that are not declared in your `reana.yaml`
+as either input or output files. This can free up a significant amount of
+space, but please be very careful to use this command only after you really
+declared all your expected outputs in `reana.yaml`!
+
+For example, with the following content:
+
+```console
+$ cat reana.yaml
+...
+outputs:
+  files:
+    - statanalysis/fitresults/limit.png
+```
+
+running the `prune` command:
 
 ```console
 $ reana-client prune -w reana-demo-root6-roofit
 ==> SUCCESS: The workspace has been correctly pruned.
 ```
 
-By default, the command only deletes files that are neither inputs nor outputs.
-However, you can use the `--include-inputs` and `--include-outputs` flags to include these files
-in the deletion process as well.
+would mercilessly delete all files except for
+`statanalysis/fitresults/limit.png`.
+
+Note that you can use the command-line options `--include-inputs` and
+`--include-outputs` to instruct REANA to optionally remove your declared inputs
+and outputs as well.
+
+!!! warning
 
-It's important to note that running this command will permanently delete the files from your workspace,
-so make sure you have backed up any files you want to keep before running the command.
+    It is important to underline once more that running the `prune` command
+    will permanently delete any file from your workspace which is not declared
+    up front amongst expected `outputs` files. If you are unsure, please make
```

Comment:
Wouldn't it be better to also mention `inputs` files among the ones that are not deleted?